production.ini

```diff
@@ -13,7 +13,9 @@ pyramid.debug_notfound = false
 pyramid.debug_routematch = false
 pyramid.default_locale_name = en
 
-sqlalchemy.url = sqlite:///%(here)s/learing_journal.sqlite
+# sqlalchemy.url = sqlite:///%(here)s/learing_journal.sqlite
+# sqlalchemy.url = postgres://vlsxgagisvzhkx:h6yNw5aEYIPSyapUYzoZ1fkqMn@ec2-54-235-104-63.compute-1.amazonaws.com:5432/d8m6stckvjgohv
+sqlalchemy.url = postgres://prflcwuvwuhtwz:rI5AYLSpblscpfStbpQG4Grbyy@ec2-54-221-201-244.compute-1.amazonaws.com:5432/dd8tcb4l3pm8cj
```

Comment:
You don't need to keep this here. The point of using `os.environ` in your `__init__.py` is to bypass the need for this keyword in your production.ini. You can just ask Heroku directly for whatever is attached to its `DATABASE_URL` environment variable.